app/controllers/hrs_controller.rb

```diff
@@ -0,0 +1,27 @@
+class HrsController < ApplicationController
+    skip_before_action :authenticate_user!, only: [:new, :create]
+
+    def show
+        @hr = Hr.find(params[:id])
```

Comment:
Why Find?

Reply:
sorry for that,I have fixed by using find_by now.